trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -58,10 +58,17 @@ def VLM(conditions,settings,geometry):
         fineness.nose                          [Unitless]
         fineness.tail                          [Unitless]
         
-       settings.number_spanwise_vortices       [Unitless]
-       settings.number_chordwise_vortices      [Unitless]
+       settings.number_spanwise_vortices       [Unitless]  <---|
```

Comment:
Make a note in the assumptions above that the user can input either individual discretizations or universal. English text is sometimes a little more inviting

Reply:
done